modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -407,6 +407,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
                 );
                 if (!empty($QCExists)) {
                     $updateWhere['FileID'] = $curFileID;
+                    Utility::nullifyEmpty($updateSet, 'QCStatus');
```

Comment:
This will only apply to the `if` loop, what about the `else` loop when you are inserting on line 423?